access_seats_test.go

```diff
@@ -0,0 +1,19 @@
+package cloudflare
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestUpdateAccessUserSeatWithMissingUID(t *testing.T) {
+	setup()
+	defer teardown()
+
+	_, err := client.UpdateAccessUserSeat(context.Background(), testAccountRC, UpdateAccessUserSeatParams{})
+	assert.EqualError(t, err, "missing required access seat UID")
+
+	_, err = client.UpdateAccessUserSeat(context.Background(), testZoneRC, UpdateAccessUserSeatParams{})
+	assert.EqualError(t, err, "missing required access seat UID")
```

Comment:
I kind of feel that if the zoneRC is used it should error but I could not see evidence of this pattern.

Reply:
example: https://github.com/cloudflare/cloudflare-go/blob/eff3c6531617a30c1bc1d07628dab365673a2d30/waiting_room.go#L562-L564